pkg/storage/chunk/client/azure/blob_storage_client_test.go

```diff
@@ -90,7 +90,7 @@ func Test_Hedging(t *testing.T) {
 	}{
 		{
 			"delete/put/list are not hedged",
-			3,
+			4,
```

Comment:
Can we add a comment that the one additional request comes from the `GetProperties()` call when initialising the client?